.buildkite/scripts/steps/code_coverage/reporting/uploadStaticSite.sh

```diff
@@ -4,18 +4,24 @@ set -euo pipefail
 
 xs=("$@")
 
-uploadPrefix="gs://elastic-bekitzur-kibana-coverage-live/"
+# TODO: Safe to remove this after 2024-03-01 (and usages below)
```

Comment:
Can we follow up with an issue for tracking?

Reply:
Present: https://github.com/elastic/kibana/issues/175904